scripts/masterCron.ts

```diff
@@ -86,6 +88,42 @@ const getNextFriday = (currentExpiry: number) => {
   return friday8am;
 };
 
+function generateTokenSet(tokens: Array<object>) {
+  // reference: https://github.com/opynfinance/opyn-tokenlist/blob/master/opyn-v1.tokenlist.json
+  let tokenJSON = {
+    name: "Ribbon oTokens",
+    logoURI: "https://i.imgur.com/u5z1Ev2.png",
+    keywords: ["defi", "option", "opyn", "ribbon"],
+    //convert to something like 2021-09-08T10:51:49Z
+    timestamp: moment.format().toString().slice(0, -6) + "Z",
+    version: { major: 1, minor: 0, patch: 0 },
+    tokens: tokens,
+  };
+
+  return JSON.stringify(tokenJSON);
+}
+
+async function pushTokenListToGit(tokenlist: string, fileName: string) {
+  const options: Partial<SimpleGitOptions> = {
+    baseDir: process.cwd(),
+    binary: "git",
+    maxConcurrentProcesses: 6,
+  };
+
+  // when setting all options in a single object
+  const git: SimpleGit = simpleGit(options);
+
+  fs.writeFile(`../ribbon-token-list/${fileName}`, tokenlist);
+
+  await git
+    .cwd("../ribbon-token-list")
```

Comment:
setting current working directory (cwd) to other repo 
https://github.com/steveukx/git-js#changing-the-working-directory-examples